backend/development.ini.sample

```diff
@@ -1,267 +1,373 @@
+; ldap_tls = False
+## If True, LDAP own tracim group managment (not available for now!)
+; ldap_group_enabled = False
+## User auth token validity in seconds (used to interfaces like web calendars)
 user.auth_token.validity = 604800
-# user reset_password token validity (default to 900s -> 15 minutes)
+## user reset_password token validity (default to 900s -> 15 minutes)
 user.reset_password.validity = 900
 
+####
+# IHM
+####
 
-session.type = file
-session.data_dir = %(here)s/sessions_data
-session.lock_dir = %(here)s/sessions_lock
-session.key = session_key
-session.secret = change_this_value_please!
-session.save_accessed_time = True
-session.cookie_expires = 604800
-session.timeout = 604800
-session.cookie_on_exception = true
-### Mail
+### Website config ###
+## title of website, used in email
+website.title = %(basic_setup.website_title)s
+# The following base_url is used for links and icons
+# integrated in the email notifcations, it's the frontend config.
+website.base_url = %(basic_setup.website_base_url)s
+## If config not provided, it will be extracted from website.base_url
+; website.server_name = localhost
+
+### Frontend ###
+frontend.serve = True
+# You can set dist folder of tracim frontend. by default, system
+# will try to get it automatically according to tracim repository
+# organisation.
+; frontend.dist_folder_path = /home/user/tracim/frontend/dist
+
+### Color ###
+# check for color.json file in tracim, check by default in tracim parent
+# dir of backend.
+## you can set a specific file path here
+; color.config_file_path = /home/user/tracim/color.json
+
+### Webdav ###
+# webdav url for client
+# url can be extended like localhost/webdav
+# in this case, you have to create your own proxy behind this url.
+# Do not set http:// prefix.
+; wsgidav.client.base_url = localhost:<WSGIDAV_PORT>
+
+####
+# EMAIL-COMMON
+####
+
+## shortcut for template_dir path
+email.template_dir = %(here)s/tracim_backend/templates/mail
+
+### Email sending configuration ###
+# processing_mode may be sync or async,
+# if async is choosen, you need also to run mail_notifier daemon
+# see README for more info.
+email.processing_mode = sync
+## with async, please also configure redis below
+; email.async.redis.host = localhost
+; email.async.redis.port = 6379
+; email.async.redis.db = 0
 
+####
+# EMAIL-NOTIFICATION
+####
+
+## available choice: True or False
 email.notification.activated = False
-# email.notification.log_file_path = /tmp/mail-notifications.log
+## You can change file_path for mail-notifications.log
+; email.notification.log_file_path = /tmp/mail-notifications.log
+
+### SMTP ###
+email.notification.smtp.server = your_smtp_server
+email.notification.smtp.port = 25
+email.notification.smtp.user = your_smtp_user
+email.notification.smtp.password = your_smtp_password
+
+### Processing ###
+# processing_mode may be sync or async
+# if you use async you need additional daemon, see README for more info.
```

Comment:
Can you add daemon name ? It will be easier to find README part talking about.

Reply:
done.

Reply:
USe same sentence form as ` # if async is choosen, you need also to run mail_notifier daemon `